.github/workflows/test.yml

```diff
@@ -21,8 +21,7 @@ jobs:
     name: Setup for HA-core
     strategy:
       matrix:
-        branch: ["master", "dev"]
-    continue-on-error: ${{ matrix.branch == 'dev' }}
```

Comment:
I would sincerely have dev in as discussed earlier so we can catch stuff earlier. It doesn't block anything (other than the renovatebot things)

Reply:
Ok, added back.

Reply:
As discussed, reviewing alternatives outside of PRs for dev, let's leave it in till then